demo/full/scripts/modules/player/index.js

```diff
@@ -127,16 +126,33 @@ const PLAYER = ({ $destroy, state }, { videoElement, textTrackElement }) => {
       player.setVolume(volume);
     },
 
-    LOAD: (arg) => {
+    LOAD: (arg, nextInitOptions) => {
       dettachVideoThumbnailLoader();
+      if (JSON.stringify(initOptions) !== JSON.stringify(nextInitOptions)) {
+        player = new RxPlayer({
```

Comment:
Shouldn't we remove the previous player, unbind our previous event listeners and make sure that everything that needs to be cleaned is?